source/markdown/posts/2.md

```diff
@@ -0,0 +1,5 @@
+#### Mitchell Baker's Thoughts
+
+Mozilla’s Executive Chairwoman, Mitchell Baker, joined the UN Secretary-General’s High-Level Panel on Women’s Economic Empowerment in Costa Rica to prepare the first draft of the panel’s report which showcases programs and practices in women’s economic empowerment which have shown positive results so that interested parties could adopt these successful practices.
+
+[Continue Reading](#NEED_URL)
```

Comment:
https://blog.lizardwrangler.com/2016/07/22/update-on-the-united-nations-high-level-panel-on-womens-economic-empowerment/